src/systems/multicopter_motor_model/MulticopterMotorModel.cc

```diff
@@ -250,7 +252,8 @@ void MulticopterMotorModel::Configure(const Entity &_entity,
   }
   else
   {
-    ignerr << "Please specify a robotNamespace.\n";
+    ignwarn << "No robotNamespace set using entity name.\n";
+    this->dataPtr->robotNamespace = scopedName(_entity, _ecm);
```

Comment:
```suggestion
    this->dataPtr->robotNamespace = this->dataPtr->model.Name(_ecm);
```

This will approach will remove the leading 'model' from the topic names.

Reply:
Good point, will test this.